Maven Plugin: inconsistent logging of ClassAlreadyExistsException

Hi!

within [this commit](https://github.com/joelittlejohn/jsonschema2pojo/commit/07076aa57ba7c954f13a1cd4984569f6d0241bd6) logging of `ClassAlreadyExistsException` was added to `EnumRule.java`. This makes it inconsistent with other `*Rule.java` clasess. For instance `ObjectRule.java` does not do it ([see here](https://github.com/joelittlejohn/jsonschema2pojo/blob/master/jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/ObjectRule.java#L96)).

I am not sure if such logging of `ClassAlreadyExistsException` is desired but suddenly getting following errors only for enums after upgrade is quite confusing:
```
[INFO] --- jsonschema2pojo:1.2.1:generate (default) @ my-module ---
[ERROR] Could not create enum.
org.jsonschema2pojo.exception.ClassAlreadyExistsException: com.my.package.MyEnum
    at org.jsonschema2pojo.rules.EnumRule.createEnum (EnumRule.java:349)
    at org.jsonschema2pojo.rules.EnumRule.apply (EnumRule.java:116)
    at org.jsonschema2pojo.rules.EnumRule.apply (EnumRule.java:68)
    at org.jsonschema2pojo.rules.SchemaRule.apply (SchemaRule.java:81)
    at org.jsonschema2pojo.rules.SchemaRule.apply (SchemaRule.java:76)
    ...
```

Thanks for taking a look at this!

--- a/jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/EnumRule.java
+++ b/jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/EnumRule.java
@@ -115,7 +115,6 @@ public class EnumRule implements Rule<JClassContainer, JType> {
         try {
             _enum = createEnum(node, nodeName, container);
         } catch (ClassAlreadyExistsException e) {
-            ruleFactory.getLogger().error("Could not create enum.", e);
             return e.getExistingClass();
         }
 

